provision-vm.yml

```diff
@@ -54,7 +53,7 @@
     - { role: welcome,           tags: ['welcome'] }
     - { role: openconext-common, tags: ['eb','sr'] }
     - { role: engineblock,       tags: ['eb'     ] }
-    - { role: vm_only_bootstrap_ldap_attributes,       tags: ['ldap'     ] }
+#    - { role: vm_only_bootstrap_ldap_attributes,       tags: ['ldap'     ] }
```

Comment:
I asked the original committer and he doesn't know why he did that, he says that it seems wrong and that it's probably just by mistake.